test/e2e/aws_test.go

```diff
@@ -35,11 +46,11 @@ func newAWSHelper(awsAccessKeyID, awsSecretAccessKey string) (providerTestHelper
 	}, nil
 }
 
-func (a *awsTestHelper) makeCredentialsSecret(namespace string) *corev1.Secret {
+func (a *awsTestHelper) makeCredentialsSecret() *corev1.Secret {
```

Comment:
I don't like the idea that the helpers become more dependent on the global variables: namespace for the secret, hostedzone domain and Kube client. This makes this structure not modular, incomplete and hard to move.
So, I prefer the way it's currently done: the methods of the helper interface to get what they need as parameters.

Reply:
we created these as global variable so these cab be used directly .
moved the previous way